Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -220,16 +220,26 @@ try {
                         }
                     }
 
-                    # The PullRequestHandler workflow can have a RepoSetting called CICDPullRequestBranches, which will be used to set the branches for the workflow
                     if ($baseName -eq "PullRequestHandler") {
+                        # The PullRequestHandler workflow can have a RepoSetting called PRBuildTrigger which specifies the trigger to use for Pull Requests
+                        $triggerSection = $yaml.Get('on:/pull')
+                        $prTrigger = "pull_request_target" # Default trigger: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
+                        if ($repoSettings.Keys -contains 'PRBuildTrigger') {
+                            $prTrigger = $repoSettings.PRBuildTrigger
```

Comment:
I think it should be named CICDPullRequestTrigger to follow the naming with CICDPullRequestBranches (and placed together with that in the Settings.md)

Reply:
_CICDPullRequestTrigger_ is confusing as it's not affecting the triggers for CI/CD. It's only for the PR build.
In the same matter _CICDPullRequestBranches_ is also confusing, but it'll be breaking to rename it (without proper obsoletion).

My opinion is that it's better for new settings to be easy to use and understand, than to follow a naming convention that didn't age well.

Reply:
I tend to agree with Maria, but I don't have that strong of an opinion on it. Maybe it could just be PullRequestBranches?

Reply:
PullRequestTrigger is OK for me.
I think we need to create a mechanism for upgrading settings - maybe something we could run as part of Update AL-Go System Files, which then becomes part of the PR when upgrading.
This would also allow us to remove places in the code where we have to support different ways of having the same setting.
I will look into this soonish